micro-infra-hystrix/src/main/groovy/com/ofg/infrastructure/hystrix/EnableHystrixServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.ofg.infrastructure.hystrix;
+
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Enables support for a servlet emitting Hystrix data
+ *
+ * @see com.ofg.infrastructure.hystrix.HystrixServletConfiguration
+ *
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Import(HystrixServletConfiguration.class)
+public @interface EnableHystrixServlet {
```

Comment:
These could be part of those acceptance tests that we would run in the pipeline that we've been talking about it (in terms of checking that our micro-infra-spring libs haven't broken anything in our main app)